router/userRouter.js

```diff
@@ -11,10 +11,56 @@ import registerRules from "../validation/registerRules.js";
 
 const router = express.Router();
 
-router.get("/", getUsers);
-router.post("/add", validate(registerRules), addUser);
-router.post("/login", loginUser);
-router.put("/update", updateUser);
-router.delete("/delete", deleteUser);
+// router.get("/", getUsers);
+// router.post("/add", validate(registerRules), addUser);
+// router.post("/login", loginUser);
+// router.put("/update", updateUser);
+// router.delete("/delete", deleteUser);
+
+
+// ------ #issue5-endpoints work on 16/11/2021
+// just experimenting w/ endpoints --> this is not functional and will not run
+
+// routes to browse events without being a registered and logged in user
+router.get("/", getEventsForToday); // landing page
```

Comment:
I assume this one should be `/events`, right? Like `/user/events`

If the idea is to only return the events for today, I would actually use for example `/events/today` and I would reserve `/events` would return all of the events for the current user.

Reply:
we could do `/events/today` and `users/:userid/events/today` for the landing page, if that would make more sense. The idea is, that whenever you reload the page, come to the page logged in OR NOT (as we want most functionality accessible without having to be a registered user) you get the list of events happening today.
The main problem is: do i need all these endpoints in doubles? like 1 group of endpoints for a someone that is just browsing (like`/events/today`, `/event/:eventId`, `/events`, when using filtering ) AND for a logged in user that is browsing (`/users/:userId/events/today`, `/users/:userId/events/:eventId`, `/users/:userId/events`) ???